concepts/05 Widgets/Diagram/10 Data Binding/40 Data Binding Options.md

```diff
@@ -0,0 +1,330 @@
+      <td>A node image's URL. This option is in effect for nodes of the <i>cardWithImageOnLeft</i>, <i>cardWithImageOnTop</i>, or <i>cardWithImageOnRight</i> type.</td>
+      <td>imageUrl: "images/employees/30.png",</td>
+    </tr> 
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#itemsExpr">itemsExpr</a></td>
+      <td>A node's child items.</td>
+      <td>items: [{"Full_Name": "Brett Wade","Title": "IT Manager"}, {"Full_Name": "Barb Banks","Title": "Support Manager"}],</td>
+    </tr>
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#keyExpr">keyExpr</a></td>
+      <td>A node key. This option is required.</td>
+      <td>key: "101",</td>
+    </tr> 
+    <tr>
+    <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#leftExpr">leftExpr</a></td>
+      <td>The x-coordinate of a node's left border, in <a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/#units">units</a>.</td>
+      <td>left: 0.5,</td>
+    </tr> 
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#lockedExpr">lockedExpr</a></td>
+      <td>A value that indicates whether a node is locked.</br>Should return <i>true</i> or <i>false</i>.</td>
+      <td>locked: true,</td>
+    </tr>
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#parentKeyExpr">parentKeyExpr</a></td>
+      <td>A parent node key for a node.</td>
+      <td>key: "101",</td>
+    </tr> 
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#styleExpr">styleExpr</a></td>
+      <td>A node style.</td>
+      <td>style: { "stroke": "red" },</td>
+    </tr>
+    <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#textExpr">textExpr</a></td>
+      <td>A node text.</td>
+      <td>text: "ASP.NET",</td>
+    </tr> 
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#textStyleExpr">textStyleExpr</a></td>
+      <td>A node's text style.</td>
+      <td>textStyle: { "font-weight": "bold", "text-decoration": "underline" },</td>
+    </tr>
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#topExpr">topExpr</a></td>
+      <td>The y-coordinate of a node's top border, in <a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/#units">units</a>.</td>
+      <td>top: 0.875,</td>
+    </tr> 
+    <tr>
+      <td><a href="/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#typeExpr">typeExpr</a></td>
+      <td>The shape type for a node. The built-in shape types are shown in the <a href="/Documentation/Guide/Widgets/Diagram/Shape_Types/">Shape Types</a> section.</td>
```

Comment:
Change to:
A node's shape type. Built-in shape types are shown in the <a href="/Documentation/Guide/Widgets/Diagram/Shape_Types/">Shape Types</a> section.